### PR TITLE
chore: Add testing library commands

### PR DESCRIPTION
**Pull request by laurieontech**
Adding @testing-library/cypress for use in themes and MDX tests.

### CODE REVIEW

File: e2e-tests/mdx/cypress/support/commands.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Cypress.Commands.add(`assertRoute`, (route) => {
+  cy.url().should(`equal`, `${window.location.origin}${route}`)
```
Comment (wardpeet) [inline e2e-tests/mdx/cypress/support/commands.js:2]: We're used to do this
```suggestion
  cy.location(`pathname`).should(`eq`, route)
```

Comment (wardpeet) [reply e2e-tests/mdx/cypress/support/commands.js:2]: Also why abstract it away? It's a native supported command?

File: e2e-tests/themes/development-runtime/cypress/support/index.js [hunk 2]
```diff
@@ -1 +1,3 @@
 import "gatsby-cypress"
+import "@testing-library/cypress/add-commands"
+import "./commands"
```
Comment (pvdz) [inline e2e-tests/themes/development-runtime/cypress/support/index.js:3]: This file is missing newline at end of file (your editor can probably do this automatically..)
(warning icon wont show up in discussion tab)

Comment (laurieontech) [reply e2e-tests/themes/development-runtime/cypress/support/index.js:3]: Ya, I saw it. I'm not sure why our setup isn't catching that in pre-commit or even in warnings in the editor.

File: e2e-tests/mdx/cypress/support/commands.js (hunk 1 above)
Comment (laurieontech) [reply e2e-tests/mdx/cypress/support/commands.js:2]: We're not actually using it, so I think it's ok to remove. I brought it over from the setup the themes repo is using on instinct. I'll nuke it.

Comment (wardpeet) [reply e2e-tests/mdx/cypress/support/commands.js:2]: YAGNI :) THanks

**Comment by laurieontech** [comment]
I'm not sure what's going on here and it's not enough value add to keep poking at. Will revisit if/when needed.

**Pull request closed without merging**